plugins/scaffolder-backend/src/scaffolder/tasks/DatabaseTaskStore.ts

```diff
@@ -527,8 +527,10 @@ export class DatabaseTaskStore implements TaskStore {
   }
 
   async cleanWorkspace({ taskId }: { taskId: string }): Promise<void> {
-    await this.db<RawDbTaskRow>('tasks').where({ id: taskId }).update({
-      workspace: undefined,
+    await this.db.transaction(async tx => {
```

Comment:
I started to implement it without transactions, but the field 'workspace' still had the data inside.
I was able fix it only by explicitly wrapping it in transaction 😅 
Any thoughts how it can be done differently?